src/main/features/dashboard/views/macro/claimStatus/settlementAgreementRejected.njk

```diff
@@ -0,0 +1,30 @@
+{% macro offerRejectedForClaimantDashboard(claim) %}
+  <p>
+    {{ t('You’ve rejected an offer to settle out of court.') }}
```

Comment:
Please put this all on one line. adding line breaks between p tags can add white space to the content which can cause weird display effects.